openrarity/token/token.py

```diff
@@ -7,6 +7,19 @@
 
 
 def trait_count(values: RawToken) -> RawToken:
+    """Count the number of traits on a given token and append it as an attribute named
+    openrarity.trait_count.
+
+    Parameters
+    ----------
+    tokens : RawToken
+        Tokens to be augmented with openrarity.trait_count.
+
+    Returns
+    -------
+    RawToken
+        RawToken augmented with openrarity.trait_count.
+    """
```

Comment:
Let's rename this function `calculate_trait_count`.

Also be consistent with your other docstrings where you used ` tick marks around `openrarity.trait_count`